app/Traits/JsonSchemaAssertions.php

```diff
@@ -22,23 +22,18 @@ trait JsonSchemaAssertions
      * @param string $schemaUri
      * @param mixed $object
      * @param string $message
-     * @param bool $addFormattedErrorToMessage
      * @return void
      */
     public static function assertValidForSchema(
         string $schemaUri,
                $object,
         string $message = '',
-        bool $addFormattedErrorToMessage = true,
```

Comment:
There is zero code in Online Shopper that uses this fourth arg, IMO we're all better off with this pattern that always emits the errors on a failing assertion.